packages/gdl-frontend/pages/games/browse.js

```diff
@@ -0,0 +1,304 @@
+import { withRouter } from 'next/router';
+import { Typography } from '@material-ui/core';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+
+import { logEvent } from '../../lib/analytics';
+import type { Context } from '../../types';
+import type { intlShape } from 'react-intl';
+
+import ReadingLevelTrans from '../../components/ReadingLevelTrans';
+import { withErrorPage } from '../../hocs';
+import Layout from '../../components/Layout';
+import { Container, LoadingButton } from '../../elements/';
+import Head from '../../components/Head';
+import LevelHR from '../../components/Level/LevelHR';
+import { spacing } from '../../style/theme';
+import GridContainer from '../../components/BookGrid/styledGridContainer';
+import GameLink from '../../components/BookListSection/GameLink';
+
+import queryString from 'query-string';
+
+import type { BrowseGames, Category, ReadingLevel } from '../../gqlTypes';
+
+const PAGE_SIZE = 30;
+const INITIAL_PAGE_NUMBER = 1;
+
+const BROWSE_GAMES_QUERY = gql`
+  query BrowseGames($language: String!, $pageSize: Int, $page: Int) {
+    games_v2(language: $language, pageSize: $pageSize, page: $page) {
+      pageInfo {
+        page
+        pageSize
+        pageCount
+        hasPreviousPage
+        hasNextPage
+      }
+      results {
+        id
+        title
+        description
+        url
+        source
+        publisher
+        license
+        language
+        coverImage {
+          url
+          altText
+        }
+      }
+    }
+  }
+`;
+
+type Props = {
+  category: Category,
+  readingLevel: ReadingLevel,
+  router: {
+    query: {
+      lang: string,
+      category?: Category,
+      sort?: string
+    }
+  },
+  intl: intlShape
+};
+
+/**
+ * After graphql migration, there is indication that some users still does request with
+ * old readinglevel format, so we need to handle both old and new format.
+ * It is not simple to convert it to old format, because the values of readinglevel
+ * consist of both numberinc and string values which raise issues when using
+ * a numeric value as key property both here and in our graphql service.
+ * @param {readinglevel} level
+ */
+const parseReadingLevel = (level: string) => {
+  switch (level) {
+    case 'decodable':
+      return 'Decodable';
+    case '1':
+      return 'Level1';
+    case '2':
+      return 'Level2';
+    case '3':
+      return 'Level3';
+    case '4':
+      return 'Level4';
+    case 'read-aloud':
+      return 'ReadAloud';
+    default:
+      return level;
+  }
+};
+
+class BrowsePage extends React.Component<Props> {
+  static async getInitialProps({ query, asPath, apolloClient, req }: Context) {
+    try {
+      // Checks if client it is a client request, which happen if you direct access on url
+      const queryFromPath = queryString.parse(
+        req && req.url ? req.url.split(/\?/)[1] : asPath.split(/\?/)[1]
+      );
+
+      const parsedLevel = parseReadingLevel(queryFromPath.readingLevel);
```

Comment:
Trenger vi denne for games? Evt er å oppdatere denne så man returnerer `Games` når man får inn `games` på lik linje som `decodable`. Så slipper vi å hardkode `const readingLevel = 'Games';` i `render()`

Reply:
Nope, vi trenger den ikke. Da fikk jeg fjernet mer kode :D